submissions/adeyemiJuwon/dom/contact.js

```diff
@@ -0,0 +1,41 @@
+let submitFormContact = document.getElementById('contact-form');
+let emailContact = document.getElementById('email-contact');
+let fullName = document.getElementById('fullName');
+let subject = document.getElementById('subject');
+let message = document.getElementById('message');
+
+
+function required(field, error, event){
+    if(field.value == ""){
+        field.nextElementSibling.innerHTML = error;
+        event.preventDefault();
+        return false;
+    }else{
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+
+function emailNameValidation(field, error, Regex, event ){
```

Comment:
use `regex` instead of `Regex`. It is seeing this `Regex` as something else here.